rllib/examples/learner/multi_agent_cartpole_ppo.py

```diff
@@ -101,7 +101,7 @@ def policy_mapping_fn(agent_id, episode, worker, **kwargs):
         .training(num_sgd_iter=10)
         .multi_agent(policies=policies, policy_mapping_fn=policy_mapping_fn)
         .rl_module(_enable_rl_module_api=True)
-        .training(_enable_rl_trainer_api=True)
+        .training(_enable_learneri=True)
```

Comment:
is this supposed to be `_enable_learneri` or `_enable_learner`

Reply:
nice catch.

Reply:
the. **kwargs is stupid, it will just pass through the non-existing parameters.

Reply:
nvm it was caught by the tests :)